target/linux/ath79/image/generic-ubnt.mk

```diff
@@ -106,12 +106,12 @@ define Device/ubnt_rocket-m
 endef
 TARGET_DEVICES += ubnt_rocket-m
 
-define Device/ubnt_nano-m
+define Device/ubnt_nanostation-m
   $(Device/ubnt-xm)
-  DEVICE_TITLE := Ubiquiti Nano-M
-  SUPPORTED_DEVICES += nano-m
+  DEVICE_TITLE := Ubiquiti Nanostation M
+  SUPPORTED_DEVICES += nanostation-m
```

Comment:
@SvenRoederer Just keep there the old `SUPPORTED_DEVICES += nano-m` line for backward compatibility, as `nanostation-m` is going to be added automagicaly at https://github.com/openwrt/openwrt/blob/eb1887be93a765b4b86ecad14dd72a5a72d6870e/target/linux/ath79/image/Makefile#L63

In other words, without `nano-m` in supported devices list, the Nanostation M users on ar71xx won't be able to sysupgrade to ath79 without `-F` (force) commandline switch.

Reply:
hm, shouldn't ar71xx generally use sysupgrade -n -F ? We don't have any uci conversion scripts in place for ar71xx -> ath79 so the target switch usually entails LED / wifi breakage if the ar71xx configuration files are grandfathered into ath79.

Reply:
@ynezz but isn't for the NSM XW the same situation? ubnt-nano-m-xw (https://github.com/openwrt/openwrt/blob/master/target/linux/ar71xx/image/generic-ubnt.mk#L161)

Reply:
sadly a lot of devices are.

This ar71xx<->ath79 config sysupgrade was also the topic of a ML thread started by @ynezz 
(in a different context though!)
https://lists.openwrt.org/pipermail/openwrt-devel/2018-December/014883.html
https://lists.openwrt.org/pipermail/openwrt-devel/2018-December/014888.html
https://lists.openwrt.org/pipermail/openwrt-devel/2018-December/014899.html
https://github.com/freifunk-gluon/gluon/issues/1570

Reply:
I see this issue also for many nodes in our freifunk-network (https://github.com/freifunk-berlin/firmware/issues/662). But I think, now (before 19.0x branch) is the best time to changes names and other legacy thinks, doing it after the branch will cause even more hazzle.
When these changes have been done on the master, there is for sure enough time to fix / setup some migration.

Reply:
> but isn't for the NSM XW the same situation? ubnt-nano-m-xw

Yeah it probably is. Could you please include it into this PR as well? Thanks.

Reply:
> hm, shouldn't ar71xx generally use sysupgrade -n -F ?

As I understand it, it doesn't work right now, but it's planned before switching to ath79 and deprecating ar71xx.

Reply:
done